tests/integration/helm_opentelemetry_operator_instr_test.go

```diff
@@ -0,0 +1,144 @@
+//go:build allversions
+// +build allversions
+
+package integration
+
+import (
+	"context"
+	"fmt"
+	"testing"
+	"time"
+
+	otoperatorappsv1 "github.com/open-telemetry/opentelemetry-operator/apis/v1alpha1"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/client-go/kubernetes/scheme"
+	"sigs.k8s.io/e2e-framework/klient/k8s/resources"
+	"sigs.k8s.io/e2e-framework/klient/wait"
+	"sigs.k8s.io/e2e-framework/klient/wait/conditions"
+	"sigs.k8s.io/e2e-framework/pkg/envconf"
+	"sigs.k8s.io/e2e-framework/pkg/features"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/SumoLogic/sumologic-kubernetes-collection/tests/integration/internal/ctxopts"
+)
+
+type internalCtxKey string
+
+func Test_OpenTelemetry_Operator_Instrumentation(t *testing.T) {
+
+	const (
+		tickDuration = time.Second
+		waitDuration = time.Minute * 2
+	)
+
+	var originalNamespaceKey internalCtxKey = "originalNamespace"
+
+	installChecks := []featureCheck{
+		CheckSumologicSecret(3),
+		CheckTracesWithoutGatewayInstall,
+	}
+
+	featInstall := GetInstallFeature(installChecks)
+
+	// It is required to add v1alpha1 OT Operator Scheme to K8s Scheme
+	// https://github.com/open-telemetry/opentelemetry-operator/issues/772
+	if err := otoperatorappsv1.AddToScheme(scheme.Scheme); err != nil {
+		require.Fail(t, "failed to register scheme: %v", err)
+	}
```

Comment:
Might be better to move this to an `init` function. I think we want all of our tests to be able to use the operator CRDs. We should probably do the same for all of our CRDs, though that can happen in a separate PR.

Reply:
I will perform that change in separated PR. Is it fine for you?